### ISSUE TITLE
census: only train loss and only eval accuracy showing up in tensorboard

### ISSUE THREAD

**Issue by jcomfort4** [GoogleCloudPlatform/cloudml-samples#65]
For problems running the sample code please provide the following information.

### System information
- Mac
- 1.2
- Python 2.7

I am following the Tensorflow Census sample and running the ml-engine package locally and inspecting the tensorboard output. For loss I get the training loss, but not the eval loss and for accuracy I get the eval accuracy but not the training accuracy.

Should I be seeing both train and eval for both loss/accuracy scalars? If this is expected, how can I output both train and eval for both metrics?

**Comment by elibixby** [comment]
Going to close this. This is just a general TF question, (how do I request additional metrics) which doesn't really belong here. I'd like to keep this forum for actual issues with the sample. If you post it to StackOverflow I'll be happy to answer it. (You should also probably include relevant code snippets/links to show which sample you're working with and what you want as output).

**Comment by jcomfort4** [comment]
Sorry, I understand that this should be restricted to issues with the sample. I was not sure if it was an error in the sample code or not - in Keras, the defaults are to produce both train & acc for both the training data and the evaluation data so I was not sure if this was related to the Monitors being deprecated or just that TF has different defaults (which seems to be the case).

**Comment by elibixby** [comment]
No problem. By default only loss is logged during training because accuracy requires additional calculation (whereas loss must already be calculated for gradients). If you submit this question to StackOverflow I'll tell you how to log accuracy there ;-)

**Comment by jcomfort4** [comment]
Sorry for the delay, but if the offer still stands I posted the a related/inverse question on [stackoverflow](https://stackoverflow.com/questions/45720464/add-custom-loss-to-eval-metric-ops) (how to add loss to eval_metric_ops) as I think that I figured out how to add accuracy for training by creating a tf.summary.scalar('accuracy', accuracy) and this is a similarly related problem that I am having for creating any of my own eval_metric_ops. Thanks!

**Comment by jcomfort4** [comment]
@elibixby I haven't been able to get any responses to this other [question](https://stackoverflow.com/questions/45765996/eval-metric-ops-display-under-family-on-tensorboard) on stackoverflow. I'm looking to expand on adding loss to the eval_metric_ops while plotting them on tensorboard under the same tab/family -- is there any way to do this with eval_metric_ops?
